cli/src/command/wallet.rs

```diff
@@ -243,6 +252,27 @@ pub async fn set_node_url_command(storage_path: &Path, snapshot_path: &Path, url
     Ok(wallet)
 }
 
+pub async fn set_pow_command(
+    storage_path: &Path,
+    snapshot_path: &Path,
+    local_pow: bool,
+    worker_count: Option<usize>,
+) -> Result<Wallet, Error> {
+    let password = get_password("Stronghold password", !snapshot_path.exists())?;
+    let wallet = unlock_wallet(storage_path, snapshot_path, password).await?;
+    // Need to get the current node, so it's not removed
+    let node = wallet.client().get_node().await?;
+    let mut client_options = ClientOptions::new()
+        .with_node(node.url.as_ref())?
+        .with_local_pow(local_pow);
+    if let Some(worker_count) = worker_count {
+        client_options = client_options.with_pow_worker_count(worker_count);
+    }
+    wallet.set_client_options(client_options).await?;
```

Comment:
Wouldn't that overwrite other options ?

Reply:
With the cli wallet one can currently one set the node and that's added again